platform/android/app/src/main/kotlin/coop/polypoly/polypod/homescreen/HomeScreenViewModel.kt

```diff
@@ -21,12 +25,12 @@ enum class SectionType {
     DEVELOPER;
 
     companion object {
-        fun fromCategoryType(type: FeatureCategory): SectionType {
+        fun fromCategoryType(type: FeatureCategoryId): SectionType {
             return when (type) {
-                FeatureCategory.yourData -> YOUR_DATA
-                FeatureCategory.knowHow -> DATA_KNOW_HOW
-                FeatureCategory.tools -> TOOLS
-                FeatureCategory.developer -> DEVELOPER
+                FeatureCategoryId.yourData -> YOUR_DATA
```

Comment:
Hum, an Id is just an unique string. Can it have all these fields? Maybe you want to say `FeatureCategoryType`. ofc it's not defined here bu elsewhere, but this where I've seen it first...

Reply:
Wanted to not modify more code than needed, but I can use the FeatureCategiryId directly.